pkg/vpcmodel/grouping.go

```diff
@@ -221,8 +221,21 @@ func (g *groupedEndpointsElems) SynthesisResourceName() string {
 	return g.Name()
 }
 
+func (g *groupedEndpointsElems) GroupedEndpointsElemsNamesList() []string {
```

Comment:
```suggestion
func (g *groupedEndpointsElems) AsNamesList() []string {
```
Struct type should not be part of a its method name